src/Microsoft.PowerShell.ConsoleHost/host/msh/ProgressNode.cs

```diff
@@ -40,6 +43,11 @@ namespace Microsoft.PowerShell
             /// The node will be displayed the same as Full, plus, the whole StatusDescription and CurrentOperation will be displayed (in multiple lines if needed).
             /// </summary>
             FullPlus = 4,
+
+            /// <summary>
+            /// The node will be displayed using ANSI escape sequences.
+            /// </summary>
+            Ansi = 5,
```

Comment:
It's probably better to remove that comment since the other values are supposed to be dynamic versions of `FullPlus` when there isn't sufficient space to render while the new one is entirely different.